src/config/profiles/list.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const configure = require('../../lib/configure')
+const callbackify = require('callbackify')
+const toCamel = require('../../lib/object-to-camel')
+
+module.exports = configure(({ ky }) => {
+  return callbackify.variadic(async (options) => {
+    options = options || {}
+
+    const searchParams = new URLSearchParams(options.searchParams)
+    searchParams.set('stream-channels', true)
+
+    const res = await ky.get('config/profile/list', {
+      timeout: options.timeout,
+      signal: options.signal,
+      headers: options.headers,
+      searchParams
```

Comment:
just pass an object { 'stream-channels': true } ky will do the rest, no need to manually instantiate URLSearchParams

Reply:
I think I just copy/pasted that from `ipfs.add`..

Reply:
Hmm, you can only set strings or numbers as values if you use a plain object (e.g. no booleans), otherwise you get:

```
Error: The `searchParams` option must be either a string, `URLSearchParams` instance or an object with string and number values
```

Reply:
https://github.com/sindresorhus/ky/issues/182

Reply:
Looks like they're going to fix this in `ky@0.16.0`